src/test/java/com/aws/greengrass/logmanager/util/UnitTestLogFileHelper.java

```diff
@@ -0,0 +1,30 @@
+package com.aws.greengrass.logmanager.util;
+
+import com.aws.greengrass.logmanager.model.LogFile;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.Random;
+
+public final class UnitTestLogFileHelper {
+
+    private UnitTestLogFileHelper() { }
+
+    public static String givenAStringOfSize(int bytesNeeded) {
+        StringBuilder testStrings = new StringBuilder();
+        Random rnd = new Random();
```

Comment:
save Random as a static field in this class, it is expensive to construct random.